guides/source/caching_with_rails.md

```diff
@@ -512,13 +512,19 @@ class ProductsController < ApplicationController
 end
 ```
 
-### A note on weak ETags
-
-Etags generated by Rails are weak by default. Weak etags allow semantically equivalent responses to have the same etags, even if their bodies do not match exactly. This is useful when we don't want the page to be regenerated for minor changes in response body. If you absolutely need to generate a strong etag, it can be assigned to the header directly.
-
-```ruby
-  response.add_header "ETag", Digest::MD5.hexdigest(response.body)
-```
```

Comment:
We should also update this example to show how to generate strong ETag with `strong_etag=`